### PR TITLE
DT-2489 Add PRUM servers to IP allowlist

### PR DESCRIPTION
**Pull request by mattops**

### CODE REVIEW

File: helm_deploy/values-preprod.yaml [hunk 1]
```diff
@@ -9,6 +9,13 @@ generic-service:
     API_BASE_URL_NOMIS: "https://api-preprod.prison.service.justice.gov.uk"
     INDEX_COMPLETE_THRESHOLD: 500000
 
+  # Additional allowlist rules
+  # See also allowlist values helm_deploy/prisoner-offender-search/values.yaml
+  allowlist:
+    prum-prod-platform-1: "18.170.20.75/32"
```
Comment (Mjwillis) [inline helm_deploy/values-preprod.yaml:15]: Should be 

- 18.134.149.68
- 18.133.201.123
- 52.56.76.34

File: helm_deploy/values-dev.yaml [hunk 2]
```diff
@@ -12,6 +12,14 @@ generic-service:
     INDEX_COMPLETE_THRESHOLD: 500000
     APPLICATIONINSIGHTS_CONFIGURATION_FILE: applicationinsights.dev.json
 
+  # Additional allowlist rules
```
Comment (Mjwillis) [inline helm_deploy/values-dev.yaml:15]: This is open to internet - not needed

**Review by Mjwillis — CHANGES_REQUESTED** [review]
IP in pre-prod incorrect

File: helm_deploy/values-dev.yaml (hunk 2 above)
Comment (mattops) [reply helm_deploy/values-dev.yaml:15]: No it's not, it is IP allow-listed  - see comment, refer to charts default `values.yaml`

File: helm_deploy/values-preprod.yaml (hunk 1 above)
Comment (mattops) [reply helm_deploy/values-preprod.yaml:15]: This being preprod, are we sure you want their test env to have access to production data?

**Pull request closed without merging**